### ISSUE TITLE
Scrolling in the terminal feels yucky

### ISSUE THREAD

**Issue by julianbraha** [zed-industries/zed#10425]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Scroll in the text editor, then in terminal window. 

The refresh rate seems lower, and scrolling feels more "jank".

### Environment

Zed Preview 0.131.1 – /Applications/Zed Preview.app
macOS 14.4.1
architecture arm64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

https://github.com/zed-industries/zed/assets/11775551/d0feb4a3-2020-46c6-a954-a2ddf87cee36

**Comment by julianbraha** [comment]
This issue seems similar, but is about different scrolling speeds: https://github.com/zed-industries/zed/issues/5130

**Comment by dwexel** [comment]
yes there's no reason why zed can't have per-pixel scrolling in the terminal

**Comment by julianbraha** [comment]
> Hi there! 👋 We're working to clean up our issue tracker by closing older issues that might not be relevant anymore. If you are able to reproduce this issue in the latest version of Zed, please let us know by commenting on this issue, and we will keep it open. If you can't reproduce it, feel free to close the issue yourself. Otherwise, we'll close it in 7 days. Thanks for your help!

Yes, this is still present on the latest Zed preview (0.177.0)
